test/integration/history_int_test.go

```diff
@@ -40,6 +44,54 @@ func (suite *HistoryIntegrationTestSuite) TestHistory_History() {
 	cp.ExpectExitCode(0)
 }
 
+func (suite *HistoryIntegrationTestSuite) TestRevert_failsOnCommitNotInHistory() {
+	suite.OnlyRunForTags(tagsuite.History)
```

Comment:
This seems like the wrong location for this test. Sure it's related to history, but both this file and the tag are in my mind intended for `state history`.